index.html

```diff
@@ -2463,7 +2481,16 @@ <h2 id="AudioContext">
                 </li>
                 <li>Set the <a>rendering thread state</a> to
                 <code>suspended</code>.
+                  <div class="note">
+                    This will stop rendering.
+                  </div>
+                </li>
+                <li>If this <a>control message</a> is being run because the
```

Comment:
Phrasing seems odd. Do you mean if the control message is being run *when* the document is being unloaded?

Reply:
I rephrased with "in reaction to".